### ISSUE TITLE
Add Linear Probing (both additive and selective) PEFT methods 

### ISSUE THREAD

**Issue by anwai98** [computational-cell-analytics/peft-sam#6]
Reference: https://openreview.net/pdf?id=LVRhXa0q5r

We should think of including:

1. Attention (this especially is quite important, and would be a base for deciding on finetuning quantized models)
2.  Batch Norm fineutning
3. Bias finetuning
4. (OPTIONAL) According to the timeline if we can manage some more practically-relevant additive PEFT methods)

PS. We discuss this later once we are at a stable point with our current additive PEFT methods.
